blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -0,0 +1,119 @@
+---
+title: 3mdeb open-source contributions
+abstract: 'We are excited to share with you the open-source contributions
+           of our team. We have been actively contributing to a number of
+           open-source projects that have a significant impact on the industry.
+           Through these contributions, we have been able to help improve the
+           overall functionality and security of these projects, while also
+           gaining valuable experience and knowledge that we can apply to our
+           client work.'
+cover: /covers/image-file.png
+author: norbert.kaminski
+layout: post
+published: true
+date: 2023-04-27
+archives: "2023"
+
+tags:
+  - contribution
+  - coreboot
+  - fwupd
+  - trenchboot
+categories:
+  - Manufacturing
+
+---
+
+We are excited to share with you the open-source contributions of our team.
+We have been actively contributing to a number of open-source projects that have
+a significant impact on the industry. Through these contributions, we have been
+able to help improve the overall functionality and security of these projects,
+while also gaining valuable experience and knowledge that we can apply to our
+client work.
+
+In addition to contributing to these open-source projects, our team is also
+passionate about promoting the benefits of open-source software. We believe
+that open-source software offers numerous advantages over proprietary software,
+such as greater transparency, flexibility, and security. By contributing to
+these projects and advocating for open-source software, we aim to foster
+a culture of collaboration, innovation, and inclusivity in the technology
+industry. We are committed to making a positive impact on the community
+through our work, and we look forward to continuing our open-source
+contributions in the future.
```

Comment:
This is trivia for anyone who would like to read about our contribution. Let's respect the reader and not waste one's time.

Let's try to go straight to the point by telling a little about technologies we worked trying to master during the described period. What do those technologies mean for our customers? Why do they need those?

Reply:
This paragraph is trivial for our core audience, but we want to broaden the circle of potential readers hence the need for general information. I have added descriptions of changes that actually took place in the code.

If we want this blog to be aimed mainly at technical people then I will remove the sections that explain what each project does and the general concept of working on open-source projects

Reply:
With an additional introduction, this paragraph is acceptable.